docs/Admin.md

```diff
@@ -575,9 +575,7 @@ const Ready = () => (
 )
 
 const App = () => (
-    <Admin ready={Ready}>
-        ...
-    </Admin>
+    <Admin ready={Ready}></Admin>
```

Comment:
I don't think it makes sense. The use case is more of an admin with a children-as-function that returns nothing.

Reply:
I remove the dots because there seem confusing for the example. Like you need or may have Admin's children when using the `ready` prop.

Reply:
I've changed it to a function as a child